src/main/scala/org/allenai/pipeline/Producer.scala

```diff
@@ -169,6 +168,22 @@ class PersistedProducer[T, -A <: Artifact](
   }
 
   override def stepInfo = step.stepInfo.copy(outputLocation = Some(artifact.url))
+
+  override def withCachingDisabled = {
```

Comment:
Off-scope: Do we really really need caching? It appears to be a source a confusion for everyone, and I'm not 100% sure which purpose does it serve :)